ui/base/src/main/kotlin/org/cru/godtools/base/ui/theme/GodToolsTheme.kt

```diff
@@ -0,0 +1,16 @@
+package org.cru.godtools.base.ui.compose
+
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.lightColorScheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+
+private val GT_BLUE = Color(0xFF3BA4DB)
```

Comment:
```suggestion
private val GT_BLUE = colorResource(R.color.gt_blue)
```
You could use this to maintain the resource values.